### DISCUSSION TITLE
How can I change 3d-IDM and Controlnet models to run on a system with 16 g ram and  4g GPU?

### DISCUSSION THREAD

**Discussion by NeDa-Y** [Project-MONAI/tutorials#1832] [General]
**Please use MONAI's Discussions tab**
For questions relating to MONAI usage, please do not create an issue.

Instead, use [MONAI's GitHub Discussions tab](https://github.com/Project-MONAI/MONAI/discussions). This can be found on the MONAI repository's main page (not the tutorials) next to Issues and Pull Requests along the top.

**Comment by KumoLiu** [comment]
Hi @NeDa-Y, for autoencoder, you can use the 16g config mentioned here: https://github.com/Project-MONAI/tutorials/tree/main/generation/3d_ldm#21-3d-autoencoder-training
And for latent diffussion training, you can try to reduce the patch size and batch size mentioned here: https://github.com/Project-MONAI/tutorials/tree/main/generation/3d_ldm#22-3d-latent-diffusion-training

Thanks.
